.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request: {}
+
+concurrency:
+  group: ci-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    name: "Tests - ${{ matrix.os }} - Node.js ${{ matrix.node-version }}"
+    runs-on: "${{ matrix.os }}-latest"
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x]
```

Comment:
@quaertym note 12.x does not work and `yarn install` fails on Mac OS on fsevents installation.
I think all the dependencies needs to be updated to fix this.

Also, we may release a new major dropping support for older Node.js versions and bumping all the dependencies but it should be separate

Reply:
Sounds good. Those that prefer can stay on the older version. But I want to release the bower changes before as a minor version first.

Reply:
@quaertym agree, that totally makes sense!